### PR TITLE
Update schemas and field_selection examples

### PR DESCRIPTION
**Pull request by JonoPrest**

### CODE REVIEW

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md [hunk 1]
```diff
@@ -76,3 +76,4 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 
 
 
+
```
Comment (DZakh) [inline docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:79]: The number of empty lines increases with every PR :)

File: static/schemas/config.evm.json [hunk 2]
```diff
@@ -302,6 +288,7 @@
           "minimum": 0
         },
         "fallback_stall_timeout": {
+          "description": "If a fallback RPC is provided, the amount of time in ms to wait before kicking off the next provider",
```
Comment (DZakh) [inline static/schemas/config.evm.json:291]: Maybe worse updating the `(default: false)` in the description for rollbacks, so it's included in the patch

**Review by DZakh — APPROVED** [review]

File: static/schemas/config.evm.json (hunk 2 above)
Comment (JonoPrest) [reply static/schemas/config.evm.json:291]: I think we need to implement serde defaults then...  But yes I agree will see what I can do.

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md (hunk 1 above)
Comment (JonoPrest) [reply docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:79]: Will fix, something must be wrong with the script.

**Pull request merged**